k8s/bases/core-cc/core-cc-task-manager-configmap.yaml

```diff
@@ -85,12 +85,13 @@ data:
           - CGM_OUT
           - METADATA
         enable-export-logs: false
+        manual-upload-base-path: CORE/CC
```

Comment:
Not sure if this value should be hardcoded in the configmap. I would have preferred having them added as an environment variable in `env.properties` file and loaded via deployment file.

(applies for all processes in `k8s` directory)